docs/source/actor-rhel7-o-rhel8.md

```diff
@@ -0,0 +1,331 @@
+    },
+    "type": "Report"
+  }
+]
+```
+
+To flush all saved messages from the repository database, run `snactor messages clear`.
+
+## Writing tests for an actor
+
+[Read the tutorial](https://leapp.readthedocs.io/en/latest/unit-testing.html) for writing an running unit and component tests
+
+## Best practices
+
+Read the best practices [document](https://github.com/oamg/leapp-repository/blob/master/docs/best-practices.md) and [Python guidelines](https://github.com/oamg/leapp-guidelines/blob/master/python-coding-guidelines.md)
+
+## Contributing actors to the Leapp project
+
+Currently all Leapp elements (i.e. actors, models, tags) are stored under a public [GitHub repository](https://github.com/oamg/leapp-repository)
+
+All new content that needs to be part of Leapp release distributed to all users should be proposed as a Pull Request in this repository.
+
+**Before submitting your work for review, make sure you have read and followed the contribution guidelines:**
+
+[Contribution guidelines for writing actors](https://github.com/oamg/leapp-repository/blob/master/CONTRIBUTING.md)
+
+This [pull request](https://github.com/oamg/leapp-repository/pull/186) gives a good example of both guidelines-driven actor implementation and thorough test coverage
+
+## Pull requests review processes
+
+Every pull request need to be reviewed by core-team members, for optimize this process please ensure your code is following some criteria:
+
+- Remove [wip][poc][tbd][donotmerge] comments and labels. WIP pull requests won't be reviewed unless specifically asked by the PR author.
+- Make sure you have covered a new actor with unit tests and these are passing if they don't and you have no idea why please seek help/guidance.
+- Test your implemented functionality doing the upgrade process before open a PR.
+- We usually take at least 3 work days to review if you need some priority in this process please let us know.
+- All criteria from the [best practices](https://github.com/oamg/leapp-repository/blob/master/docs/best-practices.md) and [Python guidelines](https://github.com/oamg/leapp-guidelines/blob/master/python-coding-guidelines.md) are expected.
+
+## FAQ
+
+### In which existing workflow phase should I place my new actor?
+
+You can decide that based on the description of the [phases](https://leapp-to.github.io/upgrades). Please note that if your actor depends on some message generated by another actor, it cannot be executed in a phase before the phase of such actor. In a similar way, if your actor produces data, it needs to be executed before the actor consuming the data.
```

Comment:
Obsoleted link to leapp-to IO pages

Reply:
Unless we have a more up-to-date information about phases I'd rather keep a link to the old one than no link at all. It gives at least some idea what to grep for in the codebase.

Reply:
Phases are listed higher (L64) with link to code containing description.
Phases in the github.io page do not match what we have in repository and will only confuse people, so I would get rid of it.